lib/jsdom/living/helpers/style-rules.js

```diff
@@ -18,6 +19,34 @@ exports.propertiesWithResolvedValueImplemented = {
     inherited: true,
     initial: "visible",
     computedValue: "as-specified"
+  },
+
+  // https://drafts.csswg.org/css2/#propdef-color
+  color: {
+    inherited: true,
+    initial: {
+      "a:visited:active": "rgb(255, 0, 0)",
+      "a:active": "rgb(255, 0, 0)",
+      "a:visited": "rgb(85, 26, 139)",
+      "a:link": "rgb(0, 0, 238)",
+      mark: "rgb(0, 0, 0)",
+      body: "rgb(0, 0, 0)"
+    },
```

Comment:
I think this behavior of the color property's observed value might actually be due to the UA stylesheet. Does JSDOM implement a UA stylesheet? If so, I can transfer these over there (if they don't already exist).

The [color prop spec](https://drafts.csswg.org/css2/#propdef-color) states that the initial value "depends on user agent," but I've seen a few folks suggest that the true initial value behind the UA styles is `rgb(0, 0, 0)`. If we move these defaults to a UA stylesheet, then the initial value here probably becomes `rgb(0, 0, 0)`, and we can remove the special handling for object-type initial values from `getSpecifiedValue` too.

Reply:
We do implement the UA stylesheet. It is in https://github.com/jsdom/jsdom/blob/master/lib/jsdom/browser/default-stylesheet.js . Indeed, these should not be in the algorithms, but instead derived from there. But, several of these are already specified there...